lib/slide_hub/image.rb

```diff
@@ -1,7 +1,7 @@
 require 'rubygems'
 require 'rmagick'
 
-module Oss
+module SlideHub
   class Image
     def thumbnail(src, dest)
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'w' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)